### DISCUSSION TITLE
Use libcurl in Dev-C++(MinGW)

### DISCUSSION THREAD

**Discussion by Sean537** [curl/curl#13431] [Q&A] [answered]
How can I use curl in Dev-Cpp(version 5.11, MinGW 4.9.2, ISO C++11)? Does curl support it? I want to have a try.
Maybe it needs a curl's DLL and some head files, and the`-lcurl` compiler directives. But I haven't made it yet(I don't know much about these operations).
Can anyone help me? Thanks.
My email address is [wushaoquan666@outlook.com](mailto:wushaoquan666@outlook.com)

**Comment by jay** [comment, marked as answer]
> How can I use curl in Dev-Cpp(version 5.11, MinGW 4.9.2, ISO C++11)? Does curl support it?

Yes. Please read the documentation for your IDE. You will need to add curl's include path, lib path and add -lcurl to the libs. Also, since you are on Windows, if you want to use a static libcurl instead of the libcurl DLL then you would need to add -DCURL_STATICLIB to the preprocessor definitions.

For example let's take the [official curl for Windows](https://curl.se/windows/) 64-bit build curl-8.7.1_7-win64-mingw.zip unzipped to c:\foo. The lib folder would be c:\foo\curl-8.7.1_7-win64-mingw\lib and the include folder would be c:\foo\curl-8.7.1_7-win64-mingw\include. Your application will need the libcurl DLL which is in c:\foo\curl-8.7.1_7-win64-mingw\bin
